src/python/pants/jvm/resolve/coursier_setup.py

```diff
@@ -54,6 +56,24 @@
     """
 )
 
+COURSIER_WRAPPER_SCRIPT = textwrap.dedent(
+    """\
+    set -eux
```

Comment:
This change will fix the output for using coursier to fetch artifacts, but the other case still remains: https://github.com/pantsbuild/pants/blob/118c51f3e56df7c5aecc20d1c9e1216e8f9e2a9b/src/python/pants/jvm/jdk_rules.py#L70-L163

The `coursier java-home` command is run before ~every invoke of a tool as part of consuming the `JdkSetup`. That is in the instance that was being stripped by `src/python/pants/jvm/compile.py` (and which also effects `test`, `fmt`, `lint`, etc).

I should have put repro instructions on the ticket: sorry. But to repro, can use `./pants fmt testprojects/src/jvm/::`

Reply:
OK, that would explain why I wasn't seeing it in Coursier fetch land.

@stuhood Is this PR otherwise OK? If so, I'll remove the `setrlimit` filtering and tackle in a subsequent PR.